system/ee/legacy/helpers/multibyte_helper.php

```diff
@@ -61,53 +61,16 @@ function ee_get_encoding($encoding)
 
 	function ee_mb_strlen( $str, $encoding = null ) {
 
-		if ( null === $encoding ) {
-
-			$encoding = ee()->config->item('charset') ?: 'utf8';
-
-		}
+		$encoding = ee_get_encoding($encoding);
 
-		// If standard encoding, strlen works just fine
-		if ( ! in_array( strtolower($encoding), array( 'utf8', 'utf-8' ) ) ) {
+		if ('CP850' === $encoding || 'ASCII' === $encoding) {
 
-			return strlen( $str );
+			return \strlen($s);
```

Comment:
Single character variable names no bueno.

Reply:
Updated in latest commits.